creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -95,6 +98,13 @@ class SnapbackSM {
     // Toggle to switch logs
     this.debug = true
 
+    // Start at a random userId to avoid biased processing of early users
+    this.lastProcessedUserId = this.randomIntFromIntervalInclusive(
+      0,
+      10_000_000
+    )
+    this.usersPerJob = this.nodeConfig.get('snapbackUsersPerJob')
```

Comment:
I understand the need for limiting the number of users per job. but would capping at usersPerJob potentially skip certain users? this is my understanding of a hypothetical scenario:
- 100 users in system
- this.randomIntFromIntervalInclusive() returns 100
- snapback would process 1 user
- next iteration would start at id = 100
- ... eventually only process 0 users?

and do these two class variables conflict with the modulo base logic?

Reply:
synced offline about how the modulo base logic is used as the fallback when querying DNs that don't support pagination. also about how CN would eventually send a `prev_user_id` that's so high it returns 0 users, and then it would start again at `user_id`=0 for the next queued job